CHANGELOG.adoc

```diff
@@ -37,6 +37,16 @@ generated OpenAPI spec.
 * Trait-based API definitions. See https://docs.rs/dropshot_endpoint/latest/dropshot_endpoint/attr.api_description.html[the documentation] for details.
 * https://github.com/oxidecomputer/dropshot/pull/1049[#1049] Added `HttpResponse::status_code()`
 
+=== Breaking Changes
```

Comment:
FYI it looks like this got mismerged.  It belongs in the "Unreleased changes" section I think.